web-app/js/portal/cart/Downloader.js

```diff
@@ -103,21 +103,21 @@ Portal.cart.Downloader = Ext.extend(Ext.util.Observable, {
     _downloadAsynchronously: function(collection, downloadUrl, params) {
         log.debug('downloading asynchronously', downloadUrl);
 
+        this.params = params;
         Ext.Ajax.request({
             url: downloadUrl,
             scope: this,
-            params: params,
             success: this._onAsyncDownloadRequestSuccess,
```

Comment:
@dnahodil Do you know of a nicer way to bind parameters to this success function? Something along the lines of... `this._onAsyncDownloadRequestSuccess.apply(this, params)`?